packages/end-to-end/cypress/e2e/specs/02-farm_people.js

```diff
@@ -0,0 +1,144 @@
+describe.only('Farm People', () => {
+  let users;
+  let translation;
+  let roles;
+
+  beforeEach(() => {
+    // Load the users fixture before the tests
+    cy.fixture('e2e-test-users.json').then((loadedUsers) => {
+      users = loadedUsers;
+      const user = users[Cypress.env('USER')];
+
+      // Load the locale fixture by reusing translations file
+      cy.fixture('../../../webapp/public/locales/' + user.locale + '/translation.json').then(
+        (data) => {
+          // Use the loaded data
+          translation = data;
+
+          cy.visit('/');
+          cy.get('[data-cy=email]', { timeout: 60 * 1000 }).should('exist');
+          cy.get('[data-cy=continue]').should('exist');
+          cy.get('[data-cy=continue]').should('be.disabled');
+          cy.get('[data-cy=continueGoogle]').should('exist');
+          cy.loginOrCreateAccount(
+            user.email,
+            user.password,
+            user.name,
+            user.language,
+            translation['SLIDE_MENU']['CROPS'],
+          );
+        },
+      );
```

Comment:
This block of code seems to be shared across all the different test files. Could we abstract this to a custom command?
I'm thinking that this block here is actually what's reused between tests, while `loginOrCreateAccount`, `addFarm` and `onboardCompleteQuestions` are not reused in isolation, but just in the scope of this block. I'd suggest making those last three regular functions instead, and having this whole "user/farm setup" be the reusable command. Thoughts?